packages/select/src/vaadin-select-value-button.js

```diff
@@ -14,14 +14,14 @@ registerStyles(
       min-width: 0;
     }
 
-    ::slotted(vaadin-item) {
+    ::slotted(:not([slot])) {
```

Comment:
Is there a reason to use `:not([slot])` in the selector? `::slotted(*)` should be enough

Reply:
There are also `prefix` and `suffix` slots in the value button which are not meant to be affected by this rule. Although, they are probably not used in fact. If that being so, then `::slotted(*)` should be enough.

Reply:
Right, maybe then the selector is correct as it is then, even though select doesn't internally use the slots.